gtdynamics/dynamics/ChainDynamicsGraph.h

```diff
@@ -0,0 +1,113 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file ChainDynamicsGraph.h
+ * @brief Builds a chain dynamics graph from a Robot object.
+ * @author Dan Barladeanu
+ */
+
+#pragma once
+
+
+//#include "gtdynamics/dynamics/OptimizerSetting.h"
+#include "gtdynamics/dynamics/DynamicsGraph.h"
+#include "gtdynamics/dynamics/Chain.h"
+#include <gtdynamics/optimizer/EqualityConstraint.h>
+#include "gtdynamics/factors/ContactDynamicsMomentFactor.h"
+#include "gtdynamics/factors/WrenchFactor.h"
+#include <gtdynamics/factors/ContactHeightFactor.h>
+
+namespace gtdynamics {
+
+using gtsam::Vector3;
+using gtsam::Vector3_;
+using gtsam::Vector6;
+using gtsam::Vector6_;
+using gtsam::Double_;
+using gtsam::Pose3;
+using gtsam::Point3;
+using gtsam::Rot3;
+using gtsam::Key;
+using gtsam::NonlinearFactorGraph ;
```

Comment:
You can remove all these using statements by just declaring `using namespace gtsam;`.